lighthouse/config/test.py

```diff
@@ -19,8 +19,19 @@
 
 REPORTS_DIR = "tests/data/reports"
 
-MLWH_CONN_STRING = "root@localhost"
+WAREHOUSES_RO_CONN_STRING = "root:root@localhost"
 ML_WH_DB = "unified_warehouse_test"
+EVENTS_WH_DB = "event_warehouse_test"
 
-MLWH_RW_CONN_STRING = "root:root@localhost"
+WAREHOUSES_RW_CONN_STRING = "root:root@localhost"
 MLWH_LIGHTHOUSE_SAMPLE_TABLE = "lighthouse_sample"
```

Comment:
As mentioned in slack, we should probably look at standardising the default database connection details, or else consistently pull it from the environment.

Reply:
As discussed, will look at this as a follow up release

Reply:
https://github.com/sanger/lighthouse/pull/177